valohai_cli/commands/yaml/step.py

```diff
@@ -130,7 +130,7 @@ def create_or_update_requirements(project: Project) -> None:
             requirements_lines = list(requirements)
 
     if not any('valohai-utils' in line for line in requirements_lines):
-        requirements_lines.append('valohai-utils\n')
+        requirements_lines.append('\nvalohai-utils\n')
```

Comment:
Maybe something like this, so we don't inadvertently add a blank line _if_ the file actually did end with a newline:
```suggestion
        if requirements_lines and "\n" not in requirements_lines[-1]:
            # last line didn't end with a newline, so add one now
            requirements_lines.append('\n')
        requirements_lines.append('valohai-utils\n')
```